controllers/endpoints.controller.js

```diff
@@ -0,0 +1,6 @@
+const { getEndpointsData } = require('../models/endpoints.model');
+
+exports.getApiInfo = (req, res) => {
+    const endpointsData = getEndpointsData();
+    res.json(endpointsData);
```

Comment:
Further to my comment in the test, you can change this to send an object with a key and value 🙂